kong/plugins/ddtrace/agent_writer.lua

```diff
@@ -1,21 +1,34 @@
 local resty_http = require("resty.http")
 local encoder = require("kong.plugins.ddtrace.msgpack_encode")
 local table = require("table")
+local Queue = require("kong.tools.queue")
 
 local agent_writer_methods = {}
 local agent_writer_mt = {
     __index = agent_writer_methods,
 }
 
+local queue_conf = {
+    name = "ddtrace",
+    log_tag = "ddtrace.writer",
+    max_batch_size = 100,
+    max_coalescing_delay = 2,
+    max_entries = 10000,
+    -- Intake max payload allowed is 5MiB
+    max_bytes = 5 * 8388608,
```

Comment:
```suggestion
    max_bytes = 5 * 1024 * 1024,
```

`5 * 8388608` bytes is 40MiB (`5 * 8388608` bits would actually be  5MiB)